package.json

```diff
@@ -40,12 +40,14 @@
     "node": ">=7"
   },
   "dependencies": {
-    "aws-sdk": "^2.669.0",
     "dynamodb-data-types": "^3.0.1",
     "dynamodb-update-expression": "^0.1.21",
     "dynamodb-wrapper": "^1.4.1",
     "ramda": "^0.27.0"
   },
+  "peerDependencies": {
+    "aws-sdk": "^2.669.0"
```

Comment:
Yep, agree with @nfantone, it's a breaking change and would be nice to have it in a separate PR.

Reply:
not sure why will counts as a breaking change as is not changing any feature included in this package. It's giving us the opportunity to use the latest aws-sdk fixes without doing a release of flynamo. 
But yeah, whatever you prefer. I'll remove this.

Reply:
After realising that `flynamo` does not actually `require` `aws-sdk` anywhere, I **see no harm in moving this to `peerDependencies`**. In fact, conceptually, it should't even be a dependency (peer or otherwise) at all, if it weren't for [`dynamodb-wrapper` demanding a specific `aws-sdk` version range](https://github.com/Shadowblazen/dynamodb-wrapper/blob/master/package.json#L65).

I vote 👍.